pkg/tfgen/docs.go

```diff
@@ -914,7 +914,8 @@ func (g *generator) convertHCL(hcl string) (string, string, error) {
 		defer func() {
 			v := recover()
 			if v != nil {
-				err = fmt.Errorf("panic converting HCL to %v: %v", languageName, v)
+				cmdutil.Diag().Warningf(diag.Message("", fmt.Sprintf("failed to convert HCL example to %v", languageName)))
```

Comment:
Is the warning needed or should this just be a debug? Also, how can I see the debug messages? I can't figure out what flag to use to enable them.

Reply:
> Is the warning needed or should this just be a debug? 

I do think that this should be a warning, as it's something that requires action on our end.

> Also, how can I see the debug messages? I can't figure out what flag to use to enable them.

Honestly, I'm not certain myself.